### ISSUE TITLE
Residual Gated Dynamic Sparse Network for Gearbox Fault Diagnosis Using Multisensor Data

### ISSUE THREAD

**Issue by leemik3** [One-Day-One-Paper/One-Day-One-Paper#71]
# Residual Gated Dynamic Sparse Network for Gearbox Fault Diagnosis Using Multisensor Data
- Huang , H., Tang, B., Luo, J., Pu, H., & Zhang, K. (2021). Residual Gated Dynamic Sparse Network for Gearbox Fault Diagnosis Using Multi Sensor Data. IEEE Transactions on Industrial Informatics
---
## 문제
Gearbox(변속기) fault diagnosis

- 기존 : 한 센서의 vibration signal에 집중 
    - 정보가 한정적이고 노이즈의 영향이 크다
- 제안 : multi sensor signal 고려
    - 센서마다 fault sensitivity가 다르다. (mounted location과 noise interference 에 따라서)
    - 이로 인해 ineffective component 발생
    - 따라서 fault related feature 감지하기 위한 RGDSNet 제안

### Contributions
1. LCAU (Lightweight Channel Attention Unit)
2. GDSU (Gated Dynamic Sparse Unit
3. Combining multiple criteria  improve loss function

# 2. Proposed RGDSNet
1. Input layer
 Multisensor signal  multichannel data

2. RGDSNet
    - Backbone : residual network
    - Components : residual block (RB)
    1. LCAU
        - RB 앞에 사용
        - Fault related 된 정보로 channel의 feature extraction 수행
    2. GDSU
        - RB 내에 사용
        - 필요 없는 component 를 제거, multi channel에 선택적 mask 생성, feature fusion을 가속화

    - Softmax classification

# 3. Experimental Validation
Dataset : Drivetrain diagnostics simulator (DDS) dataset
* Drivetrain = 구동렬 = transmission = gearbox = 변속기 = 기어

### 비교 실험
1. Baseline
2. LCAU 검증 실험
    - SE - classic squeeze-and-excitation
    - ECA – SE 발전된 형태
    - LCAU – 제안하는 unit
3. GDSU 검증 실험
    - CBAM – global attention 
    - STSU - original
    - GDSU – 제안하는  module
4. 제안 모델

# 4. Future work
- Raw vibration signal 에 대해 정확도 향상
- Automatically select the optimal activation ratio based on input data
